TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -129,6 +130,17 @@ public class TrafficManagerTool
             return TransparencyToAlpha(transparency);
         }
 
+        internal NetTool NetTool {
```

Comment:
Caching is nice. The value should reset correctly on level reload, even if we don't "support" reload, we should try to do it right. Question: Does `NetTool.FirstOrDefault()` value change on level reload?

Reply:
No it doesn't change because it's vanilla tool and we need cursor texture only :relaxed: I will try second load anyways. Thanks for pointing about that

Reply:
That's what I thought. Since `TrafficManagerTool` extends `DefaultTool` which is `Monobehavior`, each Scene change (load/reload level/editor/return to menu/etc.) destroys all instantiated `MonoBehaviour` objects automatically. `DontDestroyOnLoad(gameObject)` can prevent auto-destruction, but not used in this case).